### DISCUSSION TITLE
Prestashop Module

### DISCUSSION THREAD

**Discussion by JoseManuelPedraja** [ain/smartbanner.js#809] [General]
Based on your code I have been able to create this module for Prestashop 8 and above.
I will be attentive to your updates to continue implementing them.

Thanks, without this repository I would not have created my own, thanks!!!

https://github.com/JoseManuelPedraja/smartbanner

**Comment by ain** [comment]
You're welcome @JoseManuelPedraja, and thanks for sharing here!
